arm-compute/2015-11-01-preview/swagger/containerService.json

```diff
@@ -0,0 +1,516 @@
+        "tags": [
+          "ContainerService"
+        ],
+        "operationId": "ContainerService_Get",
+        "description": "The operation to get a container service.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "containerServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the container service within the given subscription and resource group."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ContainerService"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ContainerService"
+        ],
+        "operationId": "ContainerService_Delete",
+        "description": "The operation to delete a container service.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "containerServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the container service within the given subscription and resource group."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": ""
+          },
+          "204": {
+            "description": ""
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/containerServices": {
+      "get": {
+        "tags": [
+          "ContainerService"
+        ],
+        "operationId": "ContainerService_List",
+        "description": "The operation to list container services.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
I would recommend adding the x-ms-pageable extension over here.
The structure of ContainerServiceListResult fits that bill. I understand that it does not have nextLink. This is perfectly fine. Explicitly setting the nextLink to null will not generate the nextLink property for it.
You still get the benefit of paging enumerable in C# and the value array flattened in node.js

Take a look at [storageaccounts_list] (https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2015-06-15/swagger/storage.json#L299) operation and the [StorageAccountListResult](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2015-06-15/swagger/storage.json#L654) object. They have the same scenario. Taking a look at their generated code in [C#](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/Storage/Microsoft.Azure.Management.Storage/Generated/StorageAccountsOperations.cs#L1246) and [user will directly get the array in node.js](https://github.com/Azure/azure-sdk-for-node/blob/master/test/services/storageManagement/storageManagementClient-tests.js#L139) will give you a better idea.